src/environments/environment.model.ts

```diff
@@ -3,6 +3,8 @@ import {AuthConfig} from "angular-oauth2-oidc";
 export interface Environment {
   production: boolean;
   label: string;
-  backendUrl: string;
-  authConfig: AuthConfig;
+  backendUrl?: string;
+  authConfig?: AuthConfig;
+  disableBackend: boolean;
```

Comment:
We've been wondering whether it would make more sense to drop the `disableBackend` field, and instead make it so a null `backendUrl` means that the backend is disabled. Personally I think that'd make sense since `backendUrl` is nullable.